### DISCUSSION TITLE
Babel -> IE9 will automagically work out of the box ?

### DISCUSSION THREAD

**Discussion by anjanesh** [babel/babel#12648] [Q&A] [answered]
I have a situation where I need to support IE9 and above.

If I just include `<script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>` in my HTML, will all my ES2015/ES2017 JavaScript code magically work automatically ?

No React or Angular or Vue etc - vanilla JavaScript code on a bunch of HTML files. Has jQuery.

**Comment by nicolo-ribaudo** [comment, marked as answer]
I suggest not loading the whole Babel compiler in the browser, but to transpile your JS files locally (with `@babel/core`, `@babel/cli` and `@babel/preset-env`).

You also need to include a polyfill (for example, es5-shim+es6-shim or core-js) for the new JS functions, since Babel is only about syntax.
